flink-connectors/flink-connector-mongodb/src/main/java/org/apache/flink/mongodb/table/MongodbConnectorOptions.java

```diff
@@ -0,0 +1,80 @@
+package org.apache.flink.mongodb.table;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+
+/** Options for the Mongodb connector. */
+@PublicEvolving
+public class MongodbConnectorOptions {
+
+    public static final ConfigOption<String> DATABASE = ConfigOptions.key("database".toLowerCase())
```

Comment:
All of these `toLowerCase` in the options calls are not needed and just make it harder to read.